src/swap/SwapScreen.test.tsx

```diff
@@ -99,7 +99,10 @@ const mockStoreTokenBalances = {
     priceUsd: '13.05584965485329753569',
   },
   [mockTestTokenTokenId]: {
-    ...mockTokenBalances[mockTestTokenTokenId],
```

Comment:
test token is not part of the mock token balances object actually, and other tests rely on this object so i didn't want to change it here